MAUI/Maps/vector-layers/line-layer.md

```diff
@@ -582,4 +582,110 @@ this.Content = maps;
 
 {% endtabs %}
 
-![Line shape dash array](images/line-layer/line_shape_dash_array.png)
\ No newline at end of file
+![Line shape dash array](images/line-layer/line_shape_dash_array.png)
+
+## Animation
+
+You can apply animation for the [`MapLine`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html) using the `MapLineLayer.AnimationDuration` and `MapLineLayer.AnimationEasing` properties and able to customize the animation curve and duration.
```

Comment:
You can apply animation for the [`MapLine`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html) using the `MapLineLayer.AnimationDuration` and `MapLineLayer.AnimationEasing` properties and be able to customize the animation curve and duration.

Reply:
Updated the content